### PR TITLE
Changed `useSafeArea` default value

### PR DESCRIPTION
**Pull request by mendyEdri**
## Description
CustomKeyboard had a fix some time ago, to remove the safe area (annoying grey part at the bottom).
In order to avoid migration back at the time, I've set `useSafeArea` default value to `true` although other UILib components set it to `false` by default. 
In this PR it fixed, and moved to be `false` by default

## Changelog
When using `KeyboardAccessoryView`, `useSafeArea` default value has changed from `true` to `false`, to match all other UILib's api that by default it is `false`.

### CODE REVIEW

File: lib/components/Keyboard/KeyboardTracking/KeyboardTrackingView.ios.js [hunk 1]
```diff
@@ -36,7 +36,6 @@ class KeyboardTrackingView extends PureComponent {
     useSafeArea: PropTypes.bool
   };
 
-  /** V6 should be change to default false */
   static defaultProps = {
     useSafeArea: true
```
Comment (Inbal-Tish) [inline lib/components/Keyboard/KeyboardTracking/KeyboardTrackingView.ios.js:40]: Why here it's not 'false'?

Comment (mendyEdri) [reply lib/components/Keyboard/KeyboardTracking/KeyboardTrackingView.ios.js:40]: It's a different feature, that we reverted eventually.
(The row I deleted in the spreadsheet)

**Comment by Inbal-Tish** [comment]
@mendyEdri Please check why the build is faling

File: lib/components/Keyboard/KeyboardTracking/KeyboardTrackingView.ios.js (hunk 1 above)
Comment (Inbal-Tish) [reply lib/components/Keyboard/KeyboardTracking/KeyboardTrackingView.ios.js:40]: ok

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**